.gitignore

```diff
@@ -21,3 +21,12 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+.idea/
+*.iml
+node_modules/
+/target
+/src/main/java/org/folio/rest/client/*
+/src/main/java/org/folio/rest/jaxrs/*
+/src/main/java/storage/model/*
+/src/main/resources/postgres-conf.json
```

Comment:
Sorry - unsure what .idea and *.iml are but since this is a backend module - we probably do not need node_modules. We also might not need storage/model/* since this module does not talk to DB, same with /resources/postgres-conf.json - what do you think?

We might want to add

Reply:
Hello Sobha,
.idea and .iml files are specific files for IDE that we use
and you are right we can get rid of node_modules, storage/model/ and resources/postgres-conf.json. 
Thank you for the review

Reply:
👍